outputs.tf

```diff
@@ -1,3 +1,13 @@
 output "id" {
   value = "${null_resource.default.triggers.id}"
-}
\ No newline at end of file
+}
+
+output "tags" {
+  value = "${
+    map(
+      "Name", "${null_resource.default.triggers.id}",
```

Comment:
Should we have `"Name", "${null_resource.default.triggers.name}",` instead of `"Name", "${null_resource.default.triggers.id}",`?

Because we already have 
```
output "id" {
    value = "${null_resource.default.triggers.id}"
} 
```